cmake/FindLibBfd.cmake

```diff
@@ -61,7 +78,18 @@ FIND_PACKAGE_HANDLE_STANDARD_ARGS(LibBfd "Please install the libbfd development
 mark_as_advanced(LIBBFD_INCLUDE_DIRS LIBBFD_LIBRARIES)
 
 if(${LIBBFD_FOUND})
-SET(CMAKE_REQUIRED_LIBRARIES bfd opcodes)
+find_package(LibOpcodes)
+SET(CMAKE_REQUIRED_LIBRARIES ${LIBBFD_LIBRARIES} ${LIBOPCODES_LIBRARIES})
+# libbfd.a is not statically linked with libiberty.a or libz.a so we must manually
+# do it. Furthermore, libbfd uses some libc symbols that we must manually
+# link against if we're not using static libc (which includes such symbols).
+if(STATIC_LINKING)
+  list(APPEND CMAKE_REQUIRED_LIBRARIES ${LIBIBERTY_LIBRARIES} ${LIBZ_LIBRARIES})
```

Comment:
I will have to test if this fixes the issue for building on openSUSE, but my gut instinct is that it won't -- the issue is that we have to create a dynamically-linked binary but statically link against `libbfd` (and thus `libopcodes`, `libiberty` and `zlib`). `if(STATIC_LINKING)` doesn't apply to the openSUSE case (though I'm sure it does fix an actual bug).

Reply:
I see. I think a `STATIC_BINUTILS` build flag would be reasonable. But that can go in a different PR.